Numsense.UnitTests/FrenchExample.fs

```diff
@@ -0,0 +1,256 @@
+[<InlineData(    321000, "trois-cent-vingt-et-un-mille")>]
+[<InlineData(    411416, "quatre-cent-onze-mille-quatre-cent-seize")>]
+[<InlineData(    530121, "cinq-cent-trente-mille-cent-vingt-et-un")>]
+[<InlineData(    600000, "six-cents-mille")>]
+[<InlineData(    788000, "sept-cent-quatre-vingt-huit-mille")>]
+[<InlineData(    876540, "huit-cent-soixante-seize-mille-cinq-cent-quarante")>]
+[<InlineData(    908077, "neuf-cent-huit-mille-soixante-dix-sept")>]
+[<InlineData(   1000000, "un-million")>]
+[<InlineData(   2000002, "deux-millions-deux")>]
+[<InlineData(   3040506, "trois-millions-quarante-mille-cinq-cent-six")>]
+[<InlineData(   4321000, "quatre-millions-trois-cent-vingt-et-un-mille")>]
+[<InlineData(   5004621, "cinq-millions-quatre-mille-six-cent-vingt-et-un")>]
+[<InlineData(   6982001, "six-millions-neuf-cent-quatre-vingt-deux-mille-un")>]
+[<InlineData(   7000000, "sept-millions")>]
+[<InlineData(   8000220, "huit-millions-deux-cent-vingt")>]
+[<InlineData(   9099000, "neuf-millions-quatre-vingt-dix-neuf-mille")>]
+[<InlineData(  10000000, "dix-millions")>]
+[<InlineData(  24000000, "vingt-quatre-millions")>]
+[<InlineData(  39020011, "trente-neuf-millions-vingt-mille-onze")>]
+[<InlineData(  40606100, "quarante-millions-six-cent-six-mille-cent")>]
+[<InlineData(  53000000, "cinquante-trois-millions")>]
+[<InlineData(  64000098, "soixante-quatre-millions-quatre-vingt-dix-huit")>]
+[<InlineData(  70003190, "soixante-dix-millions-trois-mille-cent-quatre-vingt-dix")>]
+[<InlineData(  80000000, "quatre-vingts-millions")>]
+[<InlineData(  80000080, "quatre-vingts-millions-quatre-vingts")>]
+[<InlineData(  80000081, "quatre-vingts-millions-quatre-vingt-un")>]
+[<InlineData(  99000099, "quatre-vingt-dix-neuf-millions-quatre-vingt-dix-neuf")>]
+[<InlineData( 100000000, "cent-millions")>]
+[<InlineData( 209000000, "deux-cent-neuf-millions")>]
+[<InlineData( 398000000, "trois-cent-quatre-vingt-dix-huit-millions")>]
+[<InlineData( 439011000, "quatre-cent-trente-neuf-millions-onze-mille")>]
+[<InlineData( 560400000, "cinq-cent-soixante-millions-quatre-cents-mille")>]
+[<InlineData( 600100900, "six-cents-millions-cent-mille-neuf-cents")>]
+[<InlineData( 600200900, "six-cents-millions-deux-cents-mille-neuf-cents")>]
+[<InlineData( 700000000, "sept-cents-millions")>]
+[<InlineData( 800116000, "huit-cents-millions-cent-seize-mille")>]
+[<InlineData( 900800007, "neuf-cents-millions-huit-cents-mille-sept")>]
+[<InlineData(1000000000, "un-milliard")>]
+[<InlineData(2121000000, "deux-milliards-cent-vingt-et-un-millions")>]
+[<InlineData(
+    System.Int32.MaxValue,
+    "deux-milliards-cent-quarante-sept-millions-quatre-cent-quatre-vingt-trois-mille-six-cent-quarante-sept")>]
+
+[<Trait("Category", "French")>]
+let ``toFrench returns correct result`` (i, expected) =
+    let actual =
+        i
+        |> Numeral.toFrench
+    expected =! actual
+    
+    let i' = 
+        actual
+        |> Numeral.tryParseFrench
+    Some i =! i'
```

Comment:
Don't you already have that property covered with `Ploeh.Numsense.NumeralProperties.tryOfFrench is the inverse of toFrench`?